utils/src/main/kotlin/LicenseFilenamePatterns.kt

```diff
@@ -97,12 +97,14 @@ object LicenseFilenamePatterns {
         directory: String,
         filenamePatterns: Collection<String>
     ): List<String> {
-        val patternsForDir = filenamePatterns.map {
+        val distinctPatterns = filenamePatterns.distinct()
```

Comment:
`distinct()` does `this.toMutableSet().toList()`, but converting the collection to a set would be sufficient here.

Reply:
See: https://github.com/oss-review-toolkit/ort/pull/3168

Reply:
Done.